core/templates/domain/exploration/exploration-id-validation.service.spec.ts

```diff
@@ -26,7 +26,8 @@ describe('Exploration id validation service', () => {
   let explorationIdValidationService:
     ExplorationIdValidationService;
   let httpTestingController: HttpTestingController;
-  let validExpResults: ExplorationSummaryBackendDict;
+  let validExpResults1: ExplorationSummaryBackendDict;
+  let validExpResults2: ExplorationSummaryBackendDict;
```

Comment:
Prefer naming variables such that it properly represents the data. Here "1" and "2" do not convey much meaning.

Reply:
Done!